Dockerfile

```diff
@@ -11,13 +11,9 @@ RUN apt-get update \
             curl ca-certificates apt-transport-https gnupg \
             build-essential \
```

Comment:
I think we can remove this too. It brings in gcc and there are always vulns in it.

Reply:
Especially provided it's purged two lines below without doing anything ;)

Reply:
I agree, have removed build-essential package now.